ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-release-1.13-postsubmits.yaml

```diff
@@ -0,0 +1,42 @@
+postsubmits:
+  cri-o/cri-o:
+  - agent: kubernetes
+    branches:
+    - ^release-1\.13$
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: branch-ci-cri-o-cri-o-release-1.13-images
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --promote
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=[images]
+        command:
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: cri-o-cri-o-release-1.13.yaml
+              name: ci-operator-4.1-configs
```

Comment:
@smarterclayton I am not sure if this should be the value of the name or if it should be something like `ci-operator-1.13-configs`.

Reply:
nvm, looks like the generator fixed it